assembly/src/main/descriptors/kapua-broker.xml

```diff
@@ -167,7 +167,11 @@
                 <include>org.apache.shiro:shiro-core</include>
 
                 <include>org.elasticsearch:elasticsearch</include>
+                <include>org.elasticsearch.client:transport</include>
                 <include>org.elasticsearch:securesm</include>
+                <include>org.slf4j:slf4j-api</include>
+                <include>org.slf4j:log4j-over-slf4j</include>
+                <include>org.apache.logging.log4j:log4j-to-slf4j</include>
```

Comment:
About logging, from my undestanding there is no alternatives.
Or you give a true log4j implementation to ES (ES throws an exception end exits at startup, if no log4j implementation is found), or you can use the bridge (using slf4j).
I'm quite sure I told about this new dependency in a previous scrum and sent you an email about that.
In any case if someone has a better solution that I didn't figure out I'm glad to change the dependencies.
https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/_using_another_logger.html